pkg/proof/querier_test.go

```diff
@@ -0,0 +1,26 @@
+package proof
+
+import (
+	"math"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_safeConvert(t *testing.T) {
+	testCases := []struct {
+		input int64
+		want  int
+	}{
+		{input: math.MinInt64, want: math.MinInt64},
```

Comment:
The safe convert implementation is correct, but this test is platform dependent, if I understand correctly, and should return an error in the general case. This is passing because `int == int64` for 64 bit systems but I don't think we can assume that everywhere, no?

If we're fine with that assumption :shipit: as only devs will run the tests and we can assume they don't have 90s computers.

Reply:
yea totally agreed, I can delete the test entirely if that would help.

Reply:
8821560e7576dbb677852f2b97194a63b4b8b1f0